launcher.sh

```diff
@@ -3,4 +3,4 @@ if [ "$(xrandr | grep ' connected' | wc -l)" -eq 1 ]; then
   export FYNE_SCALE=0.25
 fi
 
-"$HOME"/.cryo_utilities/cryo_utilities gui
```

Comment:
I need to think of a better way to rectify launching in the flatpak vs. out of the flatpak. It should be an easy thing  to solve.